.github/workflows/test-install.yml

```diff
@@ -169,12 +229,6 @@ jobs:
       run:
         .github/workflows/setup.sh
 
-    - name: Run test suite
```

Comment:
Just to check: you were planning to bring this back?

Reply:
Absolutely! 😀

Reply:
> thanks @csadorf , looks good to me.
> 
> My only comment regards the job dependencies: while I get that this makes sense from a perspective of saving resources, does it lead to a slowdown in practice?
> 
> I guess the jobs don't actually depend on each other to run successfully but rather if one job fails, then we know that the others will fail as well?
> 
> I'm ok with both approaches; just curious to know whether the total runtime differs significantly (maybe the effect is anyhow small).

Yes, the idea was to save some resources, but it is true that some of those jobs do not actually depend on each other in the way that it is configured, but rather belong to a "topical unit". I guess it would be better practice to remove those dependencies and run everything maximal parallel. I will rewire the dependencies and request another review.